test/journeys/specs/messageMeet/widget-message-meet/dataApi.js

```diff
@@ -0,0 +1,137 @@
+/* eslint-disable max-nested-callbacks */
+
+import {assert} from 'chai';
+
+import testUsers from '@ciscospark/test-helper-test-users';
+
+import {switchToMessage} from '../../../lib/menu';
+
+describe(`Widget Message Meet`, () => {
+  describe(`Data API`, () => {
+    const browserLocal = browser.select(`browserLocal`);
+
+    let mccoy, spock;
+    process.env.CISCOSPARK_SCOPE = [
+      `webexsquare:get_conversation`,
+      `Identity:SCIM`,
+      `spark:people_read`,
+      `spark:rooms_read`,
+      `spark:rooms_write`,
+      `spark:memberships_read`,
+      `spark:memberships_write`,
+      `spark:messages_read`,
+      `spark:messages_write`,
+      `spark:teams_read`,
+      `spark:teams_write`,
+      `spark:team_memberships_read`,
+      `spark:team_memberships_write`,
+      `spark:kms`
+    ].join(` `);
```

Comment:
We may still need `webexsquare.get_conversation` for some things, but I think `spark:all` should handle it. I'll try it out.